www/src/components/headCommon.astro

```diff
@@ -1,5 +1,5 @@
 <!-- Global Metadata -->
-<meta charset="utf-8" />
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

Comment:
What does this do?

Reply:
enforces the charset for super-old browsers that ignore the newer `charset` feature. on second thought maybe its safe to assume that users are not on something like ie8

will change this back, at least until there is a report of someone hitting this issue on an actual device they're using